conf/carbon.conf.example

```diff
@@ -112,6 +112,11 @@ CACHE_QUERY_PORT = 7002
 # listener. You shouldn't change this unless you know what you're doing.
 # CACHE_QUERY_BACKLOG = 1024
 
+# Set to True to enable the cache to proactively create whisper files
+# for metrics that are received but don't currently exist in the cache
+# and also do not already have an existing whisper file. Defaults to False.
+# CACHE_CREATE_WHISPER_FILES = True
```

Comment:
Does it make sense to add an `_EARLY_` or change it to `PRECREATE` or something? Right now the name implies that turning this off would disable all creates, which is not what it does.